openc3-cosmos-cmd-tlm-api/app/controllers/model_controller.rb

```diff
@@ -59,5 +68,8 @@ def destroy
     @model_class.new(name: params[:id], scope: params[:scope]).destroy
     OpenC3::Logger.info("#{@model_class.name} destroyed: #{params[:id]}", scope: params[:scope], user: username())
     head :ok
+  rescue StandardError => error
+    render json: { status: 'error', message: error.message }, status: 500
+    OpenC3::Logger.error(error.formatted)
```

Comment:
Logger errors cause messages in CmdTlmServer - Probably don't want that here.   Change to using rails logger.